python/test/t_IndependentMetropolisHastings_std.py

```diff
@@ -1,19 +1,54 @@
 #! /usr/bin/env python
+# %%
 
 import openturns as ot
 import openturns.testing as ott
-from math import pi
+import math
 
 ot.TESTPREAMBLE()
 
+#ot.RandomGenerator.SetSeed(0)
+
+# %%
+# Test independentMetropolisHastings on Beta-Binomial conjugate model
+
+# Define Beta-binomial model
+a, b, lower, upper = 1.0, 1.0, 0., 1.
+n, p = 10, 0.5
+prior = ot.Beta(a, b, lower, upper)
+model = ot.Binomial(n, p)
+
+# %%
+# Simulate data and compute analytical posterior
+X = model.getSample(1)
+posterior = ot.Beta(a + X[0,0], b + n - X[0,0], lower, upper)
+
+# %%
+# Define IMH sampler
+initialState = [p]
+imh_sampler = ot.IndependentMetropolisHastings(prior, initialState, prior, [0])
```

Comment:
In order to make this a little more meaningful, maybe use a Uniform distribution as third argument instead of the prior